src/pull-request-handler.ts

```diff
@@ -98,6 +105,18 @@ function getChecksMarkdown (pullRequestStatus: PullRequestStatus) {
     .join('\n')
 }
 
+function getCommitMessageMarkdown (pullRequestInfo: PullRequestInfo, config: Config) {
+  const commitMessage = getCommitMessage(pullRequestInfo, config)
+
+  if (commitMessage === null) {
+    return `Commit message will be auto-generated by GitHub.`
```

Comment:
I'm not sure whether we should render anything about the commit message when it isn't configured. The user doesn't seem to case for this use-case.

Reply:
Resolved in https://github.com/bobvanderlinden/probot-auto-merge/pull/439/commits/f8cc5ef5d60448344a69e4c65e70f1b1381464b4